quorum-board#197: add qlight token manager plugin (#1)
baptiste-b-pegasys/doc.goquorum: docs/configure-and-manage/manage/qlight-node.md

- `none` - the token is not refreshed (this mechanism is for development/testing purposes only).
- `external` - the refreshed token must be updated in the running qlight client process by invoking the `qlight.setCurrentToken` RPC API.
- `client-security-plugin` (default) - the client security plugin is used to periodically refresh the access token. Please see the client-security-plugin mode below.

## client-security-plugin mode

> In this mode, a Go-Quorum plugin is called when reaching the expiration date, with a configuration anticipation time.

Reviewer: In this mode, the configured Go-Quorum plugin is called when the token expiry is reached.

> In this mode, a Go-Quorum plugin is called when reaching the expiration date, with a configuration anticipation time.

Author: ```suggestion
In this mode, the configured Go-Quorum plugin is called when the token expiry is reached.
```

This plugin is configurable and you can even develop your own implementation of this plugin.

The plugins are in the ConsenSys repository and are downloaded automatically in their absence in the plugins directory.

But, you can provide your own plugin implementation: you can refer to the [use of plugin with the hello world tutorial](/tutorials/use-plugins.md) and do the same with the projects below:

> - A GoQuorum plugin is based on the Hashicorp plugin model, there is a gRPC model for the communication interaction: [ProtoBuf model](https://github.com/ConsenSys/quorum-plugin-definitions/blob/master/qlight-token-manager.proto).

Reviewer: GoQuorum plugins are based on

> - A GoQuorum plugin is based on the Hashicorp plugin model, there is a gRPC model for the communication interaction: [ProtoBuf model](https://github.com/ConsenSys/quorum-plugin-definitions/blob/master/qlight-token-manager.proto).

Author: ```suggestion
- GoQuorum plugins are based on the Hashicorp plugin model, there is a gRPC model for the communication interaction: [ProtoBuf model](https://github.com/ConsenSys/quorum-plugin-definitions/blob/master/qlight-token-manager.proto).
```
